chunk/storage_client.go

```diff
@@ -18,6 +18,10 @@ type StorageClient interface {
 	CreateTable(name string, readCapacity, writeCapacity int64) error
 	DescribeTable(name string) (readCapacity, writeCapacity int64, status string, err error)
 	UpdateTable(name string, readCapacity, writeCapacity int64) error
+
+	// For storing and retrieving objects
+	PutObject(ctx context.Context, key string, data []byte) error
+	GetObject(ctx context.Context, key string) ([]byte, error)
```

Comment:
You might consider renaming this to {Put/Get}Chunk, to make it clear what they are intended for.